src/qml/README.md

```diff
@@ -77,7 +77,8 @@ they are not needed for static builds:
 ```
 sudo apt install qml-module-qtquick2 qml-module-qtquick-controls qml-module-qtquick-controls2 qml-module-qtquick-layouts qml-module-qtquick-window2 qml-module-qt-labs-settings
 ```
-
+##### Important:
+Earlier versions of Ubuntu 21.04 require to build the [dependencies](https://github.com/bitcoin-core/gui-qml/blob/main/depends/README.md).
```

Comment:
We'll figure out the matrix of support later, but for now maybe just a note that gets across the following idea (written in a nicer way) 

```
If you're unable to install these dependencies on your system, perform a depends build
```

Reply:
Updated it.